web/profiles/custom/os2loop/modules/os2loop_upvote/src/Helper/FlagHelper.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\os2loop_upvote\Helper;
+
+use Drupal\Core\Cache\Cache;
+
+/**
+ * Flaghelper for creating top comment and count flags.
+ */
+class FlagHelper {
+
+  /**
+   * Find the top comment.
+   *
+   * If there is a comment marked as "correct answer", this is it.
+   * If there is no "correct answer", the top comment will be the most upvoted.
+   */
+  public function preprocessFlag(array &$variables) {
```

Comment:
We should rename this to `preprocessField` or create a new `preprocessField` function calling this. A comment telling that this actually `Implements hook_preprocess_field().` should also be added.